mappings/net/minecraft/block/entity/MobSpawnerBlockEntity.mapping

```diff
@@ -13,4 +13,7 @@ CLASS net/minecraft/unmapped/C_qtiirvxr net/minecraft/block/entity/MobSpawnerBlo
 		ARG 1 pos
 		ARG 2 state
 		ARG 3 blockEntity
+	METHOD m_nqofrqwu setSpawnedEntity (Lnet/minecraft/unmapped/C_ogavsvbr;Lnet/minecraft/unmapped/C_rlomrsco;)V
```

Comment:
```suggestion
	METHOD m_nqofrqwu setSpawnerEntity (Lnet/minecraft/unmapped/C_ogavsvbr;Lnet/minecraft/unmapped/C_rlomrsco;)V
```
maybe?

Reply:
I considered this, but thought it might be interpreted as "set the block entity of the spawner"

Reply:
I see. I think the fact that it takes an Entity and not a BlockEntity would be enough to know the difference

Reply:
sure, I'll go with `setSpawnerEntity` then